src/common/cm/cm_public.h

```diff
@@ -38,6 +38,7 @@ Maryland 20850 USA.
 
 void         CM_LoadMap(Str::StringRef name);
 void         CM_ClearMap();
+void CM_FreeAll();
```

Comment:
I think CM_FreeAll should be called by CM_ClearMap. External modules have no need to control these two things independently.

Reply:
Done.